### PR TITLE
[SPC] Fixup links and one more test

### PR DESCRIPTION
**Pull request by stephenmcgruer**

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@rsolomakhin - please check the test carefully to see if you're convinced it would pass muster as an SPC transaction *if* it wasn't for the payment-extension blocking. I'm not sure e.g. if I set the right values for topOrigin vs RP (we block too early for me to tell programmatically).

File: secure-payment-confirmation/authentication-cannot-bypass-spc.https.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    timeout: 60000,
+    userVerification: 'required',
+    extensions: {
+      payment: paymentInputs,
+    },
+  };
+
+  return promise_rejects_dom(t, 'NotAllowedError',
+      navigator.credentials.get({publicKey}));
+}, 'Cannot bypass SPC authentication via navigator.credentials.get');
```
Comment (rsolomakhin) [inline secure-payment-confirmation/authentication-cannot-bypass-spc.https.html:55]: "Cannot bypass SPC authentication _UI_.."?

**Review by rsolomakhin — APPROVED** [review]
Cannot find any fault with the test, except for the description. Thank you for highlighting that this test needs to be checked carefully.

File: secure-payment-confirmation/authentication-cannot-bypass-spc.https.html (hunk 1 above)
Comment (stephenmcgruer) [reply secure-payment-confirmation/authentication-cannot-bypass-spc.https.html]: Done.

**Comment by stephenmcgruer** [comment]
@foolip (cc @past ) - I'm not sure what is happening here, can you take a look? The Chrome failures look like Chrome is crashing before ever getting near the test (i.e. on startup), with GPU errors.

Notably Chrome Dev updated to `98.0.4710.4-1` today (from 97) - could something be broken in M98? Looks like its failing on `master` too: https://github.com/web-platform-tests/wpt/commits/master

**Comment by foolip** [comment]
The failures here were due to https://github.com/web-platform-tests/wpt/issues/31714, with a workaround now landed.

**Pull request merged**